rqt_mission_loader/src/rqt_mission_loader/mission_loader_widget.py

```diff
@@ -3,6 +3,7 @@
 import rospkg
 import rospy
 import time
+import sys
```

Comment:
Why was this dependency imported since it's not used anywhere in this commit ?

Reply:
@francoiscr You added this line in this commit : a6908175d95f4d49c6cc55027540a8479b32e7db
Can you confirm it can be deleted ?

Reply:
Yes, it could be deleted but I would prefer to wait for the merge since the success button requested is no longer useful since the migration to flexbe and it could contain potential errors.
@Kam0ss